### ISSUE TITLE
Doesn't Refresh Resources

### ISSUE THREAD

**Issue by BillChirico** [Tuncion/vscode-fivem-resource-manager#21]
Hello,

I'm trying to refresh my resources using this tool, but it doesn't actually refresh or restart them. I'm using localhost with a password. There's no errors and it says in VS Code that it was refreshed, but it actually isn't. 

Also, is there a way to support automatic refresh on saving a file in the resource?

Thanks

**Comment by Mathu-lmn** [comment]
After reading the code, the refresh button simply detects new resources, but it doesn't actually performs the refresh command

**Comment by BillChirico** [comment]
@Mathu-lmn Is this for the top level button or the button next to each resource that you hover over?

**Comment by Tuncion** [comment]
Hi @BillChirico, 

will test it again for myself. Did you get any errors in the logs?

**Comment by BillChirico** [comment]
> Hi @BillChirico, 
> 
> 
> 
> will test it again for myself. Did you get any errors in the logs?

No errors at all, connects fine, displays resources correctly, and when restarting it says it was successful.

**Comment by Tuncion** [comment]
Hi again,

I have tried to reproduce the error, but without success. As soon as I press the following button:
![image](https://github.com/Tuncion/vscode-fivem-resource-manager/assets/58955610/83983090-b6cf-48ed-bd0e-f3aa025ac9d4)

After that I see the following in my server console: 
![image](https://github.com/Tuncion/vscode-fivem-resource-manager/assets/58955610/0dbd2eed-50c8-44bc-9f34-bc43f4a4457d)
![image](https://github.com/Tuncion/vscode-fivem-resource-manager/assets/58955610/c264cae1-3763-4406-8e64-45f23674eab5)

My Setup was:

ip:port: localhost:30120
password: <rcon_password>

> Password is the value defined in your cfg file for convar rcon_password 

In conclusion, the RCON connection from VSCode to FxServer is not working. 
But I can't tell you why with the current information. My first thought was firewall but on localhost it is unlikely

**Comment by BillChirico** [comment]
> 
> 
> Hi again,
> 
> 
> 
> I have tried to reproduce the error, but without success. As soon as I press the following button:
> 
> ![image](https://github.com/Tuncion/vscode-fivem-resource-manager/assets/58955610/83983090-b6cf-48ed-bd0e-f3aa025ac9d4)
> 
> 
> 
> After that I see the following in my server console: 
> 
> ![image](https://github.com/Tuncion/vscode-fivem-resource-manager/assets/58955610/0dbd2eed-50c8-44bc-9f34-bc43f4a4457d)
> 
> ![image](https://github.com/Tuncion/vscode-fivem-resource-manager/assets/58955610/c264cae1-3763-4406-8e64-45f23674eab5)
> 
> 
> 
> My Setup was:
> 
> 
> 
> ip:port: localhost:30120
> 
> password: <rcon_password>
> 
> 
> 
> > Password is the value defined in your cfg file for convar rcon_password 
> 
> 
> 
> In conclusion, the RCON connection from VSCode to FxServer is not working. 
> 
> But I can't tell you why with the current information. My first thought was firewall but on localhost it is unlikely
> 
> 

Maybe it's because I wasn't providing a port and/or didn't have a recon password set in the config. I feel like in that case it should've thrown  a connection error, but it properly displayed all of my resources.

**Comment by Tuncion** [comment]
Hi @BillChirico,

you need to set the rcon password in the config, otherwise your FxServer will not open a RCON connection. The reason why it displayed all resources without rcon is because the resource data comes from endpoint /info.json (See line 24 in resources.js)

**Comment by Tuncion** [comment]
But I agree a better display when the connection does not exist would be better
